### PR TITLE
Issue49

### PR DESCRIPTION
**Pull request by savag21**
Issue49

### CODE REVIEW

**Review by mvchn — COMMENTED** [review]
There are duplicates in the code
Tests are not passed

**Review by mvchn — CHANGES_REQUESTED** [review]
TODO should be resolved

File: number_guesser/number_guess.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from random import randint
+
+LN = 50  # LN - largest number of range
+n = randint(0, LN)
```
Comment (mvchn) [inline number_guesser/number_guess.py:4]: what is 0?

File: number_guesser/number_guess.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+# TODO: computer can generate 0 also
+answer = str(input("I thought of a number from 0 to " + str(LN) + ". Do you want to guess?\n"))
```
Comment (mvchn) [inline number_guesser/number_guess.py:7]: think about "I have been thinking about"

File: number_guesser/number_guess.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+if 'yes' in answer or 'YES' in answer or 'Yes' in answer:
```
Comment (mvchn) [inline number_guesser/number_guess.py:8]: the program checks the answer 6 times, 
there is no default action, 
a user should type a long string

File: number_guesser/number_guess.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    print('OK. Input your number:')
```
Comment (mvchn) [inline number_guesser/number_guess.py:9]: there is no input, the program has print only

**Review by mvchn — CHANGES_REQUESTED** [review]
code is not clean for me

File: number_guesser/number_guess.py (hunk 4 above)
Comment (savag21) [reply number_guesser/number_guess.py:9]: Made a change

File: number_guesser/number_guess.py (hunk 2 above)
Comment (savag21) [reply number_guesser/number_guess.py:7]: Made a change

File: number_guesser/number_guess.py (hunk 1 above)
Comment (savag21) [reply number_guesser/number_guess.py:4]: Added one more constant

**Review by mvchn — CHANGES_REQUESTED** [review]
need commit with changes

**Review by mvchn — APPROVED** [review]

**Pull request merged**